tutor/templates/config/defaults.yml

```diff
@@ -2,6 +2,8 @@
 # This file includes all Tutor setting defaults. Settings that do not have a
 # default value, such as passwords, should be stored in base.yml.
 # This must be defined early
+ATLAS_BRANCH: "main"
+ATLAS_REPOSITORY: "openedx/openedx-translations"
```

Comment:
Should be "https://github.com/openedx/openedx-translations.git"

Reply:
That's how atlas works at the moment. There's an open issue to support non-GitHub repositories but that's not planned yet: 

 - https://github.com/openedx/openedx-atlas/issues/20

Reply:
OK, that's a restriction that must be documented in the Tutor docs.

Reply:
@regisb where should we add the section for Atlas docs?